ui/job-view/pushes/JobButton.jsx

```diff
@@ -80,7 +83,7 @@ export default class JobButtonComponent extends React.Component {
   }
 
   render() {
-    const { job } = this.props;
+    const { job, possiblyIntermittent } = this.props;
```

Comment:
Replacing `possiblyIntermittent` with `intermittent` sounds reasonable because the task both failed and succeeded for the push.